docs/chapter_4/lesson_12.md

```diff
@@ -0,0 +1,77 @@
+Goal:
+* Learners should understand more information about 3 modules in app_state(auth, bank, staking)
+
+# Lesson 12: Dive deeper into app_state module configuration (auth, bank, staking)
+
+app.go: Is the place where all modules for blockchain operation are aggregated.   
+
+In this lesson, I will mention 3 modules that is auth, bank and stake.
+
+## Guidelines
```

Comment:
Wrong explaination for all 3 module since the scope is genesis only. Should only care about simple explaination of 3 mods.